docs/content/guides/dagster/code-references.mdx

```diff
@@ -27,7 +27,7 @@ To complete the steps in this guide, you'll need:
 
 ---
 
-## Automatically attaching code references to asset definitions
+## Automatically attaching local file code references to asset definitions
 
 ### Assets defined in Python
 
```

Comment:
I can't comment on the line below, but we should use `PyObject` and avoid an import that goes through a private path.